controllers/indexercluster_controller.go

```diff
@@ -106,6 +106,9 @@ func (r *IndexerClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 
 // ApplyIndexerCluster adding to handle unit test case
 var ApplyIndexerCluster = func(ctx context.Context, client client.Client, instance *enterprisev3.IndexerCluster) (reconcile.Result, error) {
+	if len(instance.Spec.ClusterManagerRef.Name) > 0 {
```

Comment:
Nit: Maybe a comment here would be nice?

Reply:
Updated in the latest commit.